release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2419,6 +2420,10 @@ openshift.io/scc: hostmount-anyuid
 +
 (link:https://issues.redhat.com/browse/KATA-1373[*KATA-1373*])
 
+* There is currently a known issue that degrades the Machine Config Operator (MCO) when you attempt to create a cluster with both FIPS and `realTimeKernel`. As a workaround, modify the merge logic, which will prevent the MCO from degrading. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2096496[*BZ#2096496*])
```

Comment:
I am unsure if the second sentence in this line is a recommendation to the reader or something that is being considered as a fix for this known issue. This might leave the reader wondering if they are supposed to do something to work around this. You could update this line to provide clarity about this. If it is something for the reader to do, you could provide an additional resources reference.